### ISSUE TITLE
Serialization

### ISSUE THREAD

**Issue by urbanslug** [stchang/graph#47]
Does graph support serialization?
I'd like to write these graphs to disk and update them with time. 
Is this not possible? I can't see it in the documentation.

**Comment by stchang** [comment]
I don't think so, since they use `struct`s. Are you using [`racket/serialize`](https://docs.racket-lang.org/reference/serialization.html)?

I think you'd have to implement your own graph instance and use `prop:serializable` though tbh I don't have much experience with this.

**Comment by urbanslug** [comment]
Yes, I am using `[racket/serialize](https://docs.racket-lang.org/reference/serialization.html)` with transparent nodes for now. 
My [use case](https://github.com/urbanslug/graphite/blob/master/graphite/algorithms/variation-graph.rkt) is quite specific so I doubt my implementation will help contribute back to this project but I will try.

**Comment by stchang** [comment]
Related: https://github.com/stchang/graph/pull/36
